tools/benchmark.py

```diff
@@ -10,11 +10,14 @@
 import time
 import shutil
 from util import run, run_output, root_path, build_path
+from sys import platform
 
 # The list of the tuples of the benchmark name and arguments
 benchmarks = [("hello", ["tests/002_hello.ts", "--reload"]),
               ("relative_import", ["tests/003_relative_import.ts",
                                    "--reload"])]
+thread_count_tests = [("set_timeout", ["tests/004_set_timeout.ts",
+                                       "--reload"])]
```

Comment:
I'd rather keep it simple for now, and not have a variable number of thread count tests. I want to keep the number of benchmarks small - like 5-10 - too many gets confusing at this early stage.

Reply:
Rather than having multiple "thread count" tests, I would rather have multiple "strace" tests. In this one you're counting [`clone(2)`](http://man7.org/linux/man-pages/man2/clone.2.html), but in the future we will want to count others. For example, we might want to count the number of [`epoll_wait(2)`](http://man7.org/linux/man-pages/man2/epoll_wait.2.html) calls in an http server. Or we might want to keep track of the total number of syscalls that is done for `tests/001_hello.js` (because we don't want that to jump unexpectedly when we import some crappy 3rd party crate).